bitnami/airflow/Chart.lock

```diff
@@ -1,12 +1,12 @@
 dependencies:
 - name: redis
   repository: oci://registry-1.docker.io/bitnamicharts
-  version: 20.1.6
+  version: 20.2.1
```

Comment:
I think you don't really need to upgrade the dependencies for this change. Keep in mind that this file is updated regularly, so please don't touch it unless you really need the latests versions.

Reply:
Okay, I will revert this change.

Reply:
Should be good with [108d0cd](https://github.com/bitnami/charts/pull/30081/commits/108d0cdb99a7b481c1f4e808c265fa12c9b4652d)